### DISCUSSION TITLE
Beyond Anime

### DISCUSSION THREAD

**Discussion by stunner999** [pystardust/ani-cli#1074] [Q&A] [answered]
Is the ani-cli team looking to make something beyond Anime? like Movies?

**Comment by justchokingaround** [comment]
mayhaps

**Comment by 71zenith** [comment, marked as answer]
check out ![lobster](https://github.com/justchokingaround/lobster/tree/v4). its made by one of the maintainers

**Comment by justchokingaround** [comment]
trash

**Comment by port19x** [comment]
Check our list of [Homies](https://github.com/pystardust/ani-cli#Homies)

Most maintainers and many of our users write small web scrapers for their own enjoyment
